be/src/main/java/codesquad/issuetracker/controller/AuthController.java

```diff
@@ -22,48 +19,40 @@
+    private final TokenService tokenService;
 
     @GetMapping("/api/login")
     public ResponseMessage login(HttpServletResponse response, @RequestParam String code) {
-        Long memberId = oAuthService.authorizeForThirdParty(code);
-        Token accessToken = accessTokenProvider.createToken(String.valueOf(memberId));
-        Token refreshToken = refreshTokenProvider.createToken(String.valueOf(memberId));
-
-        tokenManager.saveRefreshTokenByMemberId(String.valueOf(memberId), refreshToken.getToken());
+        String memberId = String.valueOf(githubOAuthClient.authorizeForThirdParty(code));
+        Token accessToken = tokenService.createAccessToken(memberId);
+        Token refreshToken = tokenService.createRefreshToken(memberId);
 
         response.addHeader("access-token", accessToken.getToken());
-        response.addCookie(createCookieWithRefreshToken(refreshToken.getToken()));
+        response.setHeader("Set-Cookie", ResponseCookie
+            .from("refresh-token", refreshToken.getToken())
+            .path("/api")
+            .build()
+            .toString());
 
         return new ResponseMessage(HttpStatus.OK, "로그인이 정상적으로 처리되었습니다.");
     }
 
-    private Cookie createCookieWithRefreshToken(String refreshToken) {
-        Cookie cookie = new Cookie("refresh-token", refreshToken);
-        cookie.setPath("/api");
-        cookie.setHttpOnly(true);
-        return cookie;
-    }
-
     @PostMapping("/api/access-token/reissue")
     public ResponseMessage reissue(HttpServletRequest request, HttpServletResponse response) {
-        Token refreshToken = refreshTokenProvider.convertToObject(TokenUtils.getRefreshToken(request));
-        tokenManager.validateDurationOfRefreshToken(refreshToken.getMemberId());
-        AccessToken renewedAccessToken = accessTokenProvider.createToken(refreshToken.getMemberId());
+        Token refreshToken = tokenService.covertToRefreshTokenObject(TokenUtils.getRefreshToken(request));
+        tokenService.validateDurationOfRefreshToken(refreshToken.getMemberId());
+        Token renewedAccessToken = tokenService.createAccessToken(refreshToken.getMemberId());
         response.addHeader("access-token", renewedAccessToken.getToken());
 
         return new ResponseMessage(HttpStatus.OK, "access 토큰이 갱신되었습니다.");
     }
 
     @GetMapping("/api/logout")
     public ResponseMessage logout(HttpServletRequest request) {
-        Token accessToken = accessTokenProvider.convertToObject(TokenUtils.getAccessToken(request));
-        Token refreshToken = refreshTokenProvider.convertToObject(TokenUtils.getRefreshToken(request));
-
-        tokenManager.invalidateToken(accessToken, refreshToken);
+        Token accessToken = tokenService.covertToAccessTokenObject(TokenUtils.getAccessToken(request));
+        Token refreshToken = tokenService.covertToRefreshTokenObject(TokenUtils.getRefreshToken(request));
+        tokenService.invalidateToken(accessToken, refreshToken);
```

Comment:
컨트롤러에서 호출 할 필요가 없는, 불필요한 메소드로 보입니다. 

```suggestion
        tokenService.invalidateToken(TokenUtils.getAccessToken(request), TokenUtils.getRefreshToken(request));
```